client/tests/unit/specs/testNotificationManager.spec.ts

```diff
@@ -1,21 +1,21 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-import { Notification as MsNotification, NotificationLevel, NotificationCenter } from '@/services/notificationCenter';
+import { Notification as MsNotification, NotificationLevel, NotificationManager } from '@/services/notificationManager';
 import { vi } from 'vitest';
 import { DateTime } from 'luxon';
 import { modalController, toastController } from '@ionic/vue';
 
-describe('Notification Center', () => {
+describe('Notification Manager', () => {
   // mock ComposerTranslation
   function t(key: string): string {
     const map = new Map<string, string>([['Notification.nextButton', 'nextButton']]);
     return map.get(key) as string;
   }
   let NOTIFS: MsNotification[];
-  let center: NotificationCenter;
+  let center: NotificationManager;
```

Comment:
little detail but would be better:
```suggestion
  let notificationManager: NotificationManager;
```